conf/ConfigureFromEnv.php

```diff
@@ -139,7 +142,12 @@
 }
 
 if(defined('SS_ERROR_LOG')) {
-	SS_Log::add_writer(new SS_LogFileWriter(BASE_PATH . '/' . SS_ERROR_LOG), SS_Log::WARN, '<=');
+	$logger = Injector::inst()->get('Logger');
+	if($logger instanceof Logger) {
+		$logger->pushHandler(new StreamHandler(BASE_PATH . '/' . SS_ERROR_LOG, Logger::WARN));
```

Comment:
Ok, I don't think it blocks merge, but this isn't something I don't think I can't suggest a change to later. Maybe I'll raise a PR then.

In my idea, the subclass would be injected, so it would only rely on a yml config to refactor it out, whereas configurefromenv isn't. The use case I am thinking of "I want to inject a new handler for sites which define SS_ERROR_LOG". which currently isn't possible.